deepinterpolation/generator_collection.py

```diff
@@ -1089,85 +1101,117 @@ def __getitem__(self, index):
 
         return input_full, output_full
 
-    def get_lims_id_sample_from_index(self, index):
-        local_img = int(np.floor(index / self.nb_lims))
+    def _get_norm_parameters(self, index_frame):
```

Comment:
I actually think this must be named `__get_norm_parameters__` because that is the name in the base class